polyfill/test/PlainDate/prototype/valueOf/basic.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindate.prototype.valueof
+features: [Temporal]
+---*/
+
+const plainDate = Temporal.PlainDate.from("1963-02-13");
+
+assert.throws(TypeError, () => plainDate.valueOf(), "valueOf");
+assert.throws(TypeError, () => plainDate < plainDate, "<");
+assert.throws(TypeError, () => plainDate <= plainDate, "<=");
+assert.throws(TypeError, () => plainDate > plainDate, ">");
+assert.throws(TypeError, () => plainDate >= plainDate, ">=");
+assert.sameValue(plainDate === plainDate, true, "===");
```

Comment:
Should we add a separate test to check that `===` is object equality? That was present in the old test suite.